src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,250 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressWithNearbyTextFilter">
+      <p>Since Checkstyle 10.10.0</p>
+      <subsection name="Description" id="SuppressWithNearbyTextFilter_Description">
+        <p>
+          Filter <code>SuppressWithNearbyTextFilter</code> uses plain text to suppress
+          nearby audit events. The filter can suppress all checks which have Checker
+          as a parent module.
+        </p>
+      </subsection>
+      <subsection name="Notes" id="SuppressWithNearbyTextFilter_Notes">
+        <p>
+          Setting <code>.*</code> value to <code>nearbyTextPattern</code> property will see
+          <b>any</b> text as a suppression and will likely suppress all audit events in the file.
+          It is best to set this to a key phrase not commonly used in the file to help denote
+          it out of the rest of the file as a suppression. See the default value as an example.
+        </p>
+      </subsection>
+      <subsection name="Properties" id="SuppressWithNearbyTextFilter_Properties">
+        <div class="wrapper">
+          <table>
+            <tr>
+              <th>name</th>
+              <th>description</th>
+              <th>type</th>
+              <th>default value</th>
+              <th>since</th>
+            </tr>
+            <tr>
+              <td>nearbyTextPattern</td>
+              <td>Specify nearby text pattern to trigger filter to begin suppression.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>"SUPPRESS CHECKSTYLE (\w+)"</code></td>
+              <td>10.10.0</td>
+            </tr>
+            <tr>
+              <td>checkPattern</td>
+              <td>Specify check name pattern to suppress. Property can
+                also be a RegExp group index at <code>nearbyTextPattern</code>
+                in format of <code>$x</code> and be picked from target file.</td>
```

Comment:
may be something like this
`from line that match 'nearbyTextPattern'` ?

Reply:
Done.